docs/DeveloperGuide.adoc

```diff
@@ -800,11 +810,35 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |new user |register and account with NUS net ID |join the module community
+
+|`* * *` |new user |fill in my full name and phone number (strictly 8 numbers) |provide necessary information for the platform maintenance
+
+|`* * *` |new user |fill in optional fiels such as faculty, year of study etc |help my classmates know be better
+
+|`* * *` |user |add contacts |have more classmates to communicate with
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a person by NUS NET ID |locate details of persons without having to go through the entire list
+
+|`* * *` |user |search information in our module website based on keywords |nevigate and find the information I need quickly
+
+|`* * *` |user |view my TO-DO list |know all the my tasks easily
+
+|`* * *` |user |tick an item in the TO-DO list upon completion |focus on the tasks I have not done
+
+|`* * *` |user |save my answers of the exercises |show to tutor as proof of my learning outcome and revise before exams
+
+|`* * *` |user |update information of certain field(s) |keep my information up-to-date
+
+|`* * *` |user |see the timeline showing the learning progress of me and my teammates |make sure everyone is on track
+
+|`* * *` |user |see the weekly plan of my teammates (phase B) |to know the upcoming updates even before they send any pull requests to github repo
+
+|`* * *` |user |post my weekly plan (phase B) |to inform my teammates of upcoming updates even before I send any pull requests to github repo
```

Comment:
Seems like there are too many three stars features, re-consider if those features really critical for this app.